docs/en/Concept.md

```diff
@@ -0,0 +1,41 @@
+# Basic Concept                                
+##
+## 1 Message Model
+
+The RocketMQ message model is mainly composed of a Producer, a Broker and a Consumer.The Producer is responsible for producing messages, the Consumer is responsible for consuming messages, and the Broker is responsible for storing messages.     
+A Broker corresponds to one server during actual deployment, and each Broker can store messages from multiple topics, and messages from each Topic can be stored in a different Broker by sharding strategy.     
+A Message Queue is used to store the physical addresses of messages, and the Message addresses in each Topic are stored in multiple Message queues.The Consumer Group consists of multiple Consumer instances.    
+##  2 Producer
+ A producer is responsible for producing messages, typically by business systems.It sends messages generated by the business application systems to brokers. RocketMQ provides multiple paradigms of sending: synchronous, asynchronous,sequential and one-way.Both synchronous and asynchronous methods require the Broker to return confirmation information, while one-way sending is not required.
+## 3 Consumer
+ A Consumer responsible for consuming messages, typically the background system responsible for asynchronous consumption.A Consumer pulls messages from brokers and feeds them into application. In perspective of user application, two types of consumers are provided:Pull Consumer and Push Consumer.
```

Comment:
Either "A consumer is" or "The Consumer is" can be used here. For example:
The Consumer is responsible for consuming messages, typically a background application consuming messages asynchronously.

Similar cases below should also be checked and corrected.